diff -urN gnome-web-photo-0.4/src/Makefile.am gnome-web-photo-0.4.new/src/Makefile.am
--- gnome-web-photo-0.4/src/Makefile.am	2008-03-10 20:08:40.000000000 +0100
+++ gnome-web-photo-0.4.new/src/Makefile.am	2008-07-08 18:04:10.000000000 +0200
@@ -13,6 +13,7 @@
 	necko		\
 	pipnss		\
 	pref		\
+	thebes		\
 	uriloader	\
 	view		\
 	webbrwsr	\
@@ -37,7 +38,7 @@
 
 gnome_web_photo_CPPFLAGS = \
 	$(DEPENDENCY_CFLAGS)			\
-	$(addprefix -I$(GECKO_INCLUDE_ROOT)/,$(gecko_include_subdirs))	\
+	$(addprefix -I$(LIBXUL_INCLUDE_ROOT)/,$(gecko_include_subdirs))	\
 	-DSHARE_DIR=\"$(pkgdatadir)\"   	\
 	-DLOCALEDIR=\"$(datadir)/locale\"	\
 	-DGECKO_HOME=\"$(GECKO_HOME)\"	\
