--- gnome-web-photo-0.4/src/Makefile.am	2008-03-10 20:08:40.000000000 +0100
+++ gnome-web-photo-0.4.new/src/Makefile.am	2008-07-08 18:04:10.000000000 +0200
@@ -13,6 +13,9 @@
 	necko		\
 	pipnss		\
 	pref		\
+	qcms		\
+	string		\
+	thebes		\
 	uriloader	\
 	view		\
 	webbrwsr	\
@@ -37,7 +38,7 @@
 
 gnome_web_photo_CPPFLAGS = \
 	$(DEPENDENCY_CFLAGS)			\
-	$(addprefix -I$(GECKO_INCLUDE_ROOT)/,$(gecko_include_subdirs))	\
+	$(addprefix -I$(LIBXUL_INCLUDE_ROOT)/,$(gecko_include_subdirs))	\
 	-DSHARE_DIR=\"$(pkgdatadir)\"   	\
 	-DLOCALEDIR=\"$(datadir)/locale\"	\
 	-DGECKO_HOME=\"$(GECKO_HOME)\"	\
--- gnome-web-photo-0.9/m4/libxul.m4.new	2010-01-22 21:07:15.164217015 +0100
+++ gnome-web-photo-0.9/m4/libxul.m4	2010-08-24 01:24:06.985077917 +0300
@@ -59,7 +59,9 @@
 # I can figure this out, do it like this:
 libxul_cv_libdir="$($PKG_CONFIG --variable=sdkdir $libxul_cv_libxul_pkg)/bin"
 
-libxul_cv_includes="$($PKG_CONFIG --cflags-only-I libxul-unstable libxul)"
+libxul_cv_includes="$($PKG_CONFIG --cflags-only-I libxul)"
+
+nspr_libs="$($PKG_CONFIG --libs nspr)"
 
 AC_DEFINE([HAVE_LIBXUL],[1],[Define for libxul])
 
@@ -75,7 +77,7 @@
 
 # Can't use the value from the .pc file, since it seems buggy
 # Until I can figure it out, do this instead
-LIBXUL_LIBS="-L${libxul_cv_sdkdir}/lib -lxpcomglue_s -L${libxul_cv_sdkdir}/bin -lxul -lxpcom"
+LIBXUL_LIBS="-L${libxul_cv_sdkdir}/lib -lxpcomglue_s -L${libxul_cv_sdkdir}/bin -lxul -lxpcom ${nspr_libs}"
 
 # ***********************
 # Check for -fshort-wchar
