--- gnome-web-photo-0.9/src/Writer.cpp	2009-04-20 17:43:33.000000000 +0200
+++ gnome-web-photo-0.9/src/Writer.cpp.new	2010-01-22 20:44:49.054351295 +0100
@@ -197,7 +197,7 @@
       width = NSAppUnitsToIntPixels(cutout.width, p2a);
       height = NSAppUnitsToIntPixels(cutout.height, p2a);
 
-      rv = presShell->RenderDocument(cutout, PR_FALSE, PR_TRUE,
+      rv = presShell->RenderDocument(cutout, nsIPresShell::RENDER_IGNORE_VIEWPORT_SCROLLING,
                                      NS_RGB(255, 255, 255), context);
       if (NS_SUCCEEDED(rv)) {
         imgContext->DrawSurface(surface, gfxSize(width, height));
